wagtail/wagtailadmin/templates/wagtailadmin/pages/confirm_unpublish.html

```diff
@@ -11,20 +11,24 @@
             {% csrf_token %}
             <input type="hidden" name="next" value="{{ next }}">
             <ul class="fields">
-                {% if live_descendant_pages %}
+                {% if live_descendant_count and live_descendant_count > 0 %}
```

Comment:
Since this will always be a number, you could shorten this to: ``{% if live_descendant_count > 0  %}``